packages/generator/src/shared/UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst.ts

```diff
@@ -94,7 +91,7 @@ export function __UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst(
 
   function getV2Types() {
     return `
-  export interface ${objectDefIdentifier} extends InterfaceDefinition<"${interfaceDef.apiName}", ${interfaceDef.apiName}>, VersionBound<$ExpectedClientVersion> {
+  export interface ${objectDefIdentifier} extends InterfaceDefinition<"${interfaceDef.shortApiName}", ${interfaceDef.uniqueImportName}>, VersionBound<$ExpectedClientVersion> {
```

Comment:
Why is this `<shortApiName, uniqueImportName>` ?

Reply:
Ah, this is so later when I decide to import it as `$Import$com_foo_bar$Baz` it will work. I need to copy this style to more places.